### PR TITLE
Fixing XML documentation warnings in MRTK3 and some variable name spelling fixes, first pass

### PR DESCRIPTION
**Pull request by AMollis**
## Overview

This is the first pass at fixing XML documentation warnings.  There are still 700+ more warnings to resolve.

### CODE REVIEW

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/GridSqueezer.cs [hunk 1]
```diff
@@ -50,11 +50,11 @@ public GridLayoutGroup GridLayoutGroup
         }
 
         [SerializeField]
-        [Tooltip("The RectTransform to fit the collider onto.")]
+        [Tooltip("The AttachedRectTransform to fit the collider onto.")]
```
Comment (marlenaklein-msft) [inline UnityProjects/MRTKDevTemplate/Assets/Scripts/GridSqueezer.cs:53]: What's the reason for this change?

Comment (marlenaklein-msft) [reply UnityProjects/MRTKDevTemplate/Assets/Scripts/GridSqueezer.cs:53]: Also curious is this script used in any of the samples? I can't find anywhere it is used

File: com.microsoft.mrtk.input/Experimental/SpatialMouse/Interactor/SpatialMouseInteractor.cs [hunk 2]
```diff
@@ -19,7 +19,7 @@ namespace Microsoft.MixedReality.Toolkit.Input.Experimental
     /// This is an experimental feature. This class is early in the cycle, it has 
     /// been labeled as experimental to indicate that it is still evolving, and 
     /// subject to change over time. Parts of the MRTK, such as this class, appear 
-    /// to have a lot of value even if the details haven�t fully been fleshed out. 
+    /// to have a lot of value even if the details haven�t fully been fleshed out.
```
Comment (marlenaklein-msft) [inline com.microsoft.mrtk.input/Experimental/SpatialMouse/Interactor/SpatialMouseInteractor.cs:22]: this still looks odd...

Comment (marlenaklein-msft) [reply com.microsoft.mrtk.input/Experimental/SpatialMouse/Interactor/SpatialMouseInteractor.cs:22]: could be easier to say have not?

File: com.microsoft.mrtk.standardassets/Fonts/selawkb SDF.asset
Comment (marlenaklein-msft) [inline com.microsoft.mrtk.standardassets/Fonts/selawkb SDF.asset:1]: YES

File: com.microsoft.mrtk.uxcore/Experimental/NonNativeKeyboard/NonNativeKeyboard.cs [hunk 3]
```diff
@@ -14,11 +14,10 @@ namespace Microsoft.MixedReality.Toolkit.UX.Experimental
     /// A simple general use keyboard that provides an alternative to the native keyboard offered by each device.
     /// </summary>
     /// <remarks>
-    /// <remarks>
     /// This is an experimental feature. This class is early in the cycle, it has 
     /// been labeled as experimental to indicate that it is still evolving, and 
     /// subject to change over time. Parts of the MRTK, such as this class, appear 
-    /// to have a lot of value even if the details haven�t fully been fleshed out. 
+    /// to have a lot of value even if the details haven�t fully been fleshed out.
```
Comment (marlenaklein-msft) [inline com.microsoft.mrtk.uxcore/Experimental/NonNativeKeyboard/NonNativeKeyboard.cs:20]: what is the character we are supposed to for these?

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/GridSqueezer.cs (hunk 1 above)
Comment (AMollis) [reply UnityProjects/MRTKDevTemplate/Assets/Scripts/GridSqueezer.cs:53]: OOps...I had doe a find-and-replace to address some other "RectTransform" comments.  See RectTransformColliderFitter.cs...in this file there's a property called "AttachedRectTransform", but had comments referring to a property called "RectTransform"

File: com.microsoft.mrtk.input/Experimental/SpatialMouse/Interactor/SpatialMouseInteractor.cs (hunk 2 above)
Comment (AMollis) [reply com.microsoft.mrtk.input/Experimental/SpatialMouse/Interactor/SpatialMouseInteractor.cs:22]: Fixed....not sure how this happened.

File: com.microsoft.mrtk.uxcore/Experimental/NonNativeKeyboard/NonNativeKeyboard.cs (hunk 3 above)
Comment (AMollis) [reply com.microsoft.mrtk.uxcore/Experimental/NonNativeKeyboard/NonNativeKeyboard.cs:20]: Fixed

File: com.microsoft.mrtk.standardassets/Fonts/selawkb SDF.asset
Comment (AMollis) [reply com.microsoft.mrtk.standardassets/Fonts/selawkb SDF.asset:1]: Hmmm I"m not sure I meant to add this...

But yes, this file keeps getting converted...might as well commit it

**Review by marlenaklein-msft — DISMISSED** [review]

File: com.microsoft.mrtk.uxcore/Experimental/NonNativeKeyboard/NonNativeKeyboard.cs (hunk 3 above)
Comment (AMollis) [reply com.microsoft.mrtk.uxcore/Experimental/NonNativeKeyboard/NonNativeKeyboard.cs:20]: oops...thought I did a Find-All and Replace...  I guess I didn't...

Fixed now!

**Review by marlenaklein-msft — APPROVED** [review]
